api/server/dayTimeModel.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require("mongoose");
```

Comment:
Remove this model? We decided to not use this anymore, right?

Reply:
done